src/edu/washington/escience/myria/accessmethod/JdbcAccessMethod.java

```diff
@@ -425,6 +425,21 @@ public void dropAndRenameTables(final RelationKey oldRelation, final RelationKey
   }
 
   @Override
+  public void dropTableIfExistsCascade(final RelationKey relationKey) throws DbException {
+    switch (jdbcInfo.getDbms()) {
+      case MyriaConstants.STORAGE_SYSTEM_MYSQL:
+        execute("DROP TABLE IF EXISTS " + quote(relationKey));
```

Comment:
Sure, I can add it. I'm not even sure people use MySQL as the underlying db for Myria anymore? I thought most people used SQLite and Postgres for the most part.  I just added a MySQL condition because the other methods in ```JdbcAccessMethod.java``` referred to it too.